tools/codegen/diff_native_functions.py

```diff
@@ -0,0 +1,143 @@
+import argparse
+import os
+import requests  # TODO(jwtan): Figure out the conventional API to use within the PyTorch repository.
+import subprocess
+import sys
+import tempfile
+
+# TODO(jwtan): Figure out why we need this.
```

Comment:
This is because you are probably invoking the script as `python tools/codegen/diff_native_functions.py` and by default Python will set your PYTHONPATH to be rooted at `tools/codegen/` which is incorrect, it needs to be rooted at the root directory. If you do `python -m tools.codegen.diff_native_functions` instead you won't have this problem